build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.3.0'
+        classpath 'com.android.tools.build:gradle:2.0.0-alpha1'
```

Comment:
Why not use latest versions of Gradle?  Especially when Android Studio is
recommending it.

On Wed, Dec 2, 2015 at 11:00 AM, Ryan Seys <notifications@github.com> wrote:

> In build.gradle
> <https://github.com/googlesamples/androidtv-Leanback/pull/54#discussion_r46432474>
> :
>
> > @@ -5,7 +5,7 @@ buildscript {
> >          mavenCentral()
> >      }
> >      dependencies {
> > -        classpath 'com.android.tools.build:gradle:1.3.0'
> > +        classpath 'com.android.tools.build:gradle:2.0.0-alpha1'
>
> We don't want to depend on alpha versions in this sample
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/googlesamples/androidtv-Leanback/pull/54/files#r46432474>
> .
>